src/dummy.erl

```diff
@@ -1 +1,41 @@
 -module(dummy).
+
+-export([
+         dummy/0,
+         derp/0,
+         dumb/0,
+         nop/0,
+         no_spaces_between_commas/0
+        ]).
+
+-include("dummy.hrl"). 
+
+-define(A_bAd_Macro, 1).
+
+dummy() ->
+    dummy("dummy!").
+
+dummy(Msg) ->
+    io:format(Msg).
+
+derp() ->
+    io:format("derp").
+
+dumb() ->
+    io:format("dumb").
+
+nop() ->
+%%%%%%%%%%%%%%%% Long line comment %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Line 28 is too long: %%%%%%%%%%%%%%%% Long line comment %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%.